Integrations/ThinkstCanary/ThinkstCanary.py

```diff
@@ -62,8 +62,13 @@ def http_request(method, url, params=None):
     )
 
     if not res.ok:
-        demisto.debug(res.text)
-        return_error('Could not execute the request')
+        try:
+            res_json = res.json()
```

Comment:
the result may not be a json and may throw ValueError.
in this case you should catch it sepcificly and return the content sting.
more over, if you get another exception, you should print the exception str.